betterproto/tests/inputs/namespace_builtin_types/namespace_builtin_types.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+// Tests that messages may contain fields with names that are python types
+
+message Test {
+    // https://docs.python.org/2/library/stdtypes.html#numeric-types-int-float-long-complex
+    string int = 1;
+    string float = 2;
+    string complex = 3;
+
+    // https://docs.python.org/3/library/stdtypes.html#generator-types
+    string list = 10;
+    string tuple = 11;
+    string range = 12;
+
+    // https://docs.python.org/3/library/stdtypes.html#str
+    string str = 21;
+
+    // https://docs.python.org/3/library/stdtypes.html#bytearray-objects
+    string bytearray = 22;
+
+    // https://docs.python.org/3/library/stdtypes.html#bytes-and-bytearray-operations
+    string bytes = 23;
+
+    // https://docs.python.org/3/library/stdtypes.html#memory-views
+    string memoryview = 24;
+
+    // https://docs.python.org/3/library/stdtypes.html#set-types-set-frozenset
+    string set = 25;
+    string frozenset = 26;
+
+    // https://docs.python.org/3/library/stdtypes.html#dict
+    string map = 27;
```

Comment:
dict?

Reply:
oops 😅 forgot to invite dict to the party 🥳